metadata.tex

```diff
@@ -0,0 +1,15 @@
+%% This file contains the document metadata for this document to allow
+%% the MagicDraw template to not care about the specifics of the document
+%% that is being created.
+\title{Data Management Level 1 System Requirements}
+\author{
+Tim~Jenness
```

Comment:
I don't know what the semantics of authorship are here. If these document is effectively a copy of requirements from LSE-61, it seems like the LSE-61 authors ought to be credited here, perhaps (ie, adding Gregory)? Or maybe the names of everybody who has written one of the requirements which appear above?

(Ultimately, this is just quibbling; I'm happy to leave this as is, but it might be useful to come up with guidelines as to what the right answer is in general.)

Reply:
I assume more detailed requirements will be added and the author list will change later. I'll add Gregory. Don't ask me how many SP it took to copy these in the model. You would be shocked.